test/gpu/native/studies/shoc/EXECOPTS

```diff
@@ -0,0 +1 @@
+--output=false --passes=5
```

Comment:
I'm not sure if these `COMPOPTS` and `EXECOPTS` files will really apply to all the benchmarks we end up adding in this directory (e.g. after we add tests for sort which maybe wouldn't take a passes argument).

So we may end up having to reorganize (put each benchmark in its own subdir?). Anyway we can cross that bridge when we come to it.

Reply:
Organizing each benchmark in its own subdir would keep it clean. If we don't want to do that we could also give each file its own `.compopts` and `.execopts` file but that is a lot of repetition.